src/platform/mirplatform.pc.in

```diff
@@ -1,11 +1,10 @@
-prefix=@PREFIX@
-exec_prefix=@EXEC_PREFIX@
-libdir=@LIBDIR@
-includedir=@INCLUDEDIR@
+prefix=@CMAKE_INSTALL_PREFIX@
+libdir=${prefix}/@CMAKE_INSTALL_LIBDIR@
+includedir=${prefix}/@CMAKE_INSTALL_INCLUDEDIR@
 
 Name: mirplatform
 Description: Mir platform library
 Version: @MIR_VERSION@
 Requires: mircommon, mircore
-Libs: -L@LIBDIR@ -lmirplatform
-Cflags: -I@INCLUDEDIR@ -I@COMMON_INCLUDEDIR@
+Libs: -L${libdir} -lmirplatform
```

Comment:
I've not looked to see where the problem arises, but with this a local install gets a wrong mirplatform package config:
```plain
$ pkg-config --cflags mirplatform
-I/platform -I/common -I/usr/local/include/mircommon -I/usr/local/include/mircore
$ cat /usr/local/lib/pkgconfig/mirplatform.pc
prefix=/usr/local
libdir=/lib
includedir=/include
```

Reply:
Ah! CMake screws it up unless `@ONLY` is passed to `configure_file().`